modules/menu/dynamic_menu.c

```diff
@@ -228,51 +228,60 @@ static int call_xfer(struct re_printf *pf, void *arg)
 }
 
 
-static int call_att_xfer(struct re_printf *pf, void *arg)
+static int call_attended_xfer(struct re_printf *pf, void *arg)
 {
-	struct menu *menu = menu_get();
-	struct call *call = uag_call_find(menu->callid);
+	const struct cmd_arg *carg = arg;
+	struct call *call = menu_callcur();
+	struct call *attended_call;
+	struct pl callid = PL_INIT;
+	struct pl attended_callid = PL_INIT;
+	char attended_call_string[140], call_string[140];
```

Comment:
Pls use a dynamic buffers, and functions pl_strdup() and mem_deref().
I would also prefer short local variables.
`cidatt, cid, cidattpl, cidpl`